plugins/GSdx/GSLocalMemory.cpp

```diff
@@ -41,7 +41,7 @@
 		for(int x = _r.left; x < _r.right; x += w >> 3) \
 		{ \
 			const uint8* src = BlockPtr(_base + off->block.col[x]); \
-			uint8* dst = &_dst[x * bpp]; \
+			dst = &_dst[x * bpp]; \
```

Comment:
Hum keep your change. However, I will set the pointer to NULL when you exit the foreach loop. This way if someone reuse the pointer, you will have a direct crash rather than bad graphics.